src/opcodes/opcodes.cairo

```diff
@@ -177,4 +195,13 @@ pub mod Opcode {
     fn not_implemented(ref engine: Engine) {
         panic!("Opcode not implemented");
     }
+
+    fn opcode_max(ref engine: Engine) {
+        if engine.dstack.len() < 2 {
```

Comment:
I think this check can be removed, since the failure will be propagated form the `pop_int` operation in the future.